salsah/src/public/js/jquery.extsearch.js

```diff
@@ -747,7 +747,7 @@
 								break;
 							}
 							default: {
-								searchval.push(undefined); // must be!!!
+								searchval.push("empty"); // must be!!!
```

Comment:
Could you also just pass an empty string `""`?

Reply:
checking...